tests/unit/validators/UniqueValidatorTest.php

```diff
@@ -231,7 +231,7 @@ public function testValidateScopeNamespaceTargetClassForNewClass()
         $validator = new UniqueValidator();
 
         /** @var Profile $profileModel */
-        $profileModel = new Profile(['description' => 'profile customer 1']);
+        $profileModel = Yii::createObject(['__class' => Profile::class, 'description' => 'profile customer 1']);
```

Comment:
Why not use $this->factory, like above?

And globally, why use the factory when a simple instantiation could be enough. Is there an init() being called? (On phone right now, can't check)

Reply:
I preferred to keep previous syntax, using array to pass properties values.

Do you want that I change it to explicit version?

Reply:
Not needed for now

Reply:
In this case (in tests) I would prefer `factory->create` to have it shorter and more expressive.
In main framework code I would prefer not abuse DI where it is possible.
And I prefer NOT to use `Yii::createObject` and Yii helper in general.